rules/successful-reachouts/rule.md

```diff
@@ -0,0 +1,37 @@
+---
+type: rule
+title: Do you know to make your Linkedln reachouts stand out?
```

Comment:
@bradystroud 

- [ ] 1. Fix "LinkedIn"
- [ ] 2. I think we can remove the word "your"